crates/utils/src/email.rs

```diff
@@ -78,7 +78,12 @@ pub fn send_email(
   };
 
   // Set the creds if they exist
-  if let (Some(username), Some(password)) = (email_config.smtp_login, email_config.smtp_password) {
+  let smtp_password = match std::env::var("LEMMY_SMTP_PASSWORD") {
+    Ok(var) => Some(var),
+    Err(_) => email_config.smtp_password,
```

Comment:
Cleaner to use `unwrap_or`.

Reply:
I couldn't figure out how to use unwrap_or because it has to change from a Result to an Option (also I kind of directly copied this code from the LEMMY_DATABASE_URL implementation).

Reply:
I can do `let smtp_password = std::env::var("LEMMY_SMTP_PASSWORD").ok().or(email_config.smtp_password);
` if that's fine with you.